### ISSUE TITLE
No pokestops in the area

### ISSUE THREAD

**Issue by RiC0MD** [disdain13/PokeRoadie#21]
Seems like in the last 10-20 mins they may have broke something in the bot.  I did read today they were limiting location lookups once per 10s, so wondering if they're soft banning for looking to often?  This is confirmed 3 diff accounts, across 3 diff machines, 2 diff originating IP addresses.

**Comment by DrAshBooth** [comment]
I'm also getting the same issue "(ATTENTION) No usable PokeStops found in your area. Is your maximum distance too small?" over and over

**Comment by disdain13** [comment]
This is a known bug with the FeroxRev api. They are currently working on it. ALL bots are down right now. Niantic changed the interface today.

**Comment by disdain13** [comment]
I will have it fixed as soon as the API is updated. 8)

**Comment by jdleo** [comment]
waiting on this, good to hear. If he struggles I'm gonna hop on and try to reverse engineer it myself

**Comment by RiC0MD** [comment]
Seems like the app is doing some extended verification now of the cert its talking to, just tried to MITM attack it and the app just flat stopped at login.  Could be a PITA to crack at this point. https://www.reddit.com/r/pokemongodev/comments/4w1cvr/pokemongo_current_api_status/

**Comment by disdain13** [comment]
Update. They have been working hard on it, and they are making progress. Still a few data pieces to figure out.

**Comment by disdain13** [comment]
Update! They have completed the changes, and will be publishing a new version of the API very soon! Not sure exactly when, but very soon.

**Comment by RiC0MD** [comment]
https://github.com/keyphact/pgoapi/tree/master/pgoapi looks like they published, just hope to see it pushed to the C# library soon now.  Wish I understood Python a bit more, I'd just port it myself :/

**Comment by disdain13** [comment]
Yeah, it will be real soon. I am not a python guy myself either, or I would be all over it.

**Comment by disdain13** [comment]
But, I used this opportunity to get some real nice changes done. Added a great deal of stuff during downtime, and have nearly all the core components ready to switch over to using the new synchronized event model. This will allow me to make this a forms app, and begin the UI work I really want to do. 8)

**Comment by disdain13** [comment]
Over this downtime I added: 
1. Capture on both lure and incense pokemon, in addition to Wild.
2. Auto-usage of incense, lucky eggs, incubators, potions, and revives
3. Completed Auto deployment to team gym feature
4. Built Fully Synchronized event model and events in Logic class
5. Re-wrote Logger class into a singleton provider model
6. Added complete auto "Power-Up" system just like Evolution and Transfer
7. Made sure all pokemon related message were formatted similar
8. Added "TransferTrimTheFat" feature (read more in the wiki)
